devel/gdb/Portfile

```diff
@@ -86,6 +86,13 @@ pre-configure {
 
 build.dir ${configure.dir}
 
+#older macs require a new libcxx
+if {${os.platform} eq "darwin" && ${os.major} < 15} {
+    configure.depends_lib-append port:macports-libcxx
```

Comment:
This is wrong and gonna break builds on older OS. If you need this for `libc++`-consuming versions, please make it conditional accordingly.